web/app.go

```diff
@@ -100,8 +100,8 @@ func DefaultDependencies(config *Config) Dependencies {
 	araService := ara.NewAraService(viper.GetString("ara-addr"))
 	checksService := services.NewChecksService(araService, db)
 	subscriptionsService := services.NewSubscriptionsService(db)
-	hostsService := services.NewHostsService(consulClient)
-	hostsServiceNext := services.NewHostsNextService(db)
+	hostsService := services.NewHostsConsulService(consulClient)
```

Comment:
This naming is confusing as `hostsService` is used as `hostsConsulService` in the future and `hostsServiceNext` as `hostsService`.

Reply:
thanks like above i will clean it up in a separate PR